src/command.rs

```diff
@@ -139,7 +177,7 @@ pub async fn add(registry: RegistryUri, dependency: &str) -> miette::Result<()>
         .push(Dependency::new(registry, repository, package, version));
 
     manifest
-        .write()
+        .write(None)
```

Comment:
Can we do one method `write` and another one called `write_at` (or something along the lines) for the one taking a path instead of using a optional parameter?